<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowTheStateClass</class>
 <widget class="QWidget" name="ShowTheStateClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShowTheState</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="m_wgtTotal" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="m_wgtHead" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,6,3">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QWidget" name="wgt_empty" native="true">
           <widget class="QPushButton" name="m_btnAdd">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>10</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
           <widget class="QPushButton" name="m_btnDel">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>10</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="m_wgtTitle" native="true">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="m_labTittle">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>XXXXXXXX</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="m_labTime">
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="text">
               <string>XXX</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="m_wgtTol" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="m_labTotal">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>30</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Total:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
        <zorder>m_wgtTitle</zorder>
        <zorder>m_wgtTol</zorder>
        <zorder>wgt_empty</zorder>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="m_wgtExcel" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="MyTableView" name="m_tabView"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyTableView</class>
   <extends>QTableView</extends>
   <header>MyTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ShowTheState.qrc"/>
 </resources>
 <connections/>
</ui>
